packages/breadcrumbs/src/elements/Breadcrumb.js

```diff
@@ -15,31 +16,31 @@ import Item from '../views/Item';
 
 /**
  * High-level abstraction for basic Breadcrumb implementations. Accepts all
- * `<div>` props.
+ * `<ol>` props.
  */
 export default class Breadcrumb extends Component {
   static propTypes = {
     children: PropTypes.any
   };
 
   renderItems = items => {
-    const retVal = [];
     const total = Children.count(items);
 
-    Children.forEach(items, (item, index) => {
+    return Children.map(items, (item, index) => {
+      let retVal;
       const itemProps = {
         current: index === total - 1,
         key: index
       };
 
-      if (item.type === Item) {
-        retVal.push(cloneElement(item, itemProps));
+      if (hasType(item, Item)) {
```

Comment:
You could drop the `retVal` and just do a ternary:

```js
return hasType(item, Item) ? (
  cloneElement(item, itemProps)
) : (
  <Item {...itemProps}>{item}</Item>
);
```

Reply:
Of course. Good call.